<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SYSTEM</class>
 <widget class="QTabWidget" name="SYSTEM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统模块</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>teacher</string>
   </attribute>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>系统---教师</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="del_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_teacher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>991</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>student</string>
   </attribute>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>系统---学生</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="del_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_student">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>991</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_1">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>course</string>
   </attribute>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>系统---课程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_course">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>991</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
